src/msg.rs

```diff
@@ -98,6 +114,52 @@ pub struct PalomaMsg {
 /// Currently cc-amm provides no queries.
 #[cw_serde]
 #[derive(QueryResponses)]
-pub enum QueryMsg {}
+pub enum QueryMsg {
+    /// Returns registered chain information.
+    #[returns(ChainInfo)]
+    ChainInfo {
+        /// Chain Id.
+        chain_id: Uint256,
+    },
+
+    /// Returns pool_id number from chain and token information.
+    #[returns(Uint256)]
+    PoolId {
+        /// Chain 0 Id.
+        chain0_id: Uint256,
+        /// Chain 1 Id.
+        chain1_id: Uint256,
+        /// Token address on chain 0.
+        token0: String,
+        /// Token address on chain 1.
+        token1: String,
+    },
+    /// Returns pool information from pool_id.
+    #[returns(PoolInfo)]
+    PoolInfo {
+        /// Pool id.
+        pool_id: Uint256,
+    },
+
+    /// Returns state information.
+    #[returns(State)]
+    State {},
+
+    /// Returns liquidity queue information.
+    #[returns(QueueID)]
+    LiquidityQueue {
+        /// Pool id.
+        pool_id: Uint256,
+    },
+
+    /// Returns liquidity queue element data from pool_id and queue_id.
+    #[returns(LiquidityQueueElement)]
+    LiquidityQueueElement {
+        /// Pool id.
+        pool_id: Uint256,
```

Comment:
I know it's hard, but this cannot be the documentation for pool id.

1. You can make a newtype `struct PoolId(Uint256)` and document _that_ (what is a pool id?), then this documentation becomes a little forgivable.
2. OR You can describe here what a pool id is and what it's used for. Are liquidity elements identified by their pool id (one per pool?), the queue id? Both? &c.
3. Or you can do some combo. Newtypes are a little rust specific but they would go a long way towards clarifying this code.

(this applies everywhere there is an
```
/// X.
x
```
style item)